resources/lib/ytchannels.py

```diff
@@ -128,12 +161,20 @@ def ytchannels_main():
 			li.setArt({'icon':'%s'%channels[i][2]})
 
 			rem_uri = build_url({'mode': 'rem_channel', 'channel_id': '%s'%str(channels[i][1])})
-			li.addContextMenuItems([ (local_string(30003), 'RunPlugin(%s)'%rem_uri)])
+			move_down_uri = build_url({'mode': 'move_down', 'id': '%s'%channels[i][4]})
+			move_up_uri = build_url({'mode': 'move_up', 'id': '%s'%channels[i][4]})
+			items = []
+			items.append((local_string(30003), 'RunPlugin(%s)'%rem_uri))
+			if channels[i][3] == 1:
```

Comment:
Again, this shows the "move down" option if you only have 1 item in the list, which you can then do and the item ends up wierdly the second row. (Not sure which is responsible actually, this one or the above)

Reply:
fixed

Reply:
Cool